Package.swift

```diff
@@ -1,4 +1,4 @@
-// swift-tools-version:5.1
+// swift-tools-version:5.6
```

Comment:
Alternatively, we could add a separate `Package@swift-5.6.swift` which adds the plugin to maintain compatibility with older versions of Swift